EPRN.Portal/Controllers/WasteController.cs

```diff
@@ -150,10 +150,14 @@ public async Task<IActionResult> SubTypes(WasteSubTypesViewModel wasteSubTypesVi
         }
 
         [HttpGet]
-        [Route("/Status")]
-        public async Task<IActionResult> GetWasteRecordStatus(int journeyId)
+        public async Task<IActionResult> GetWasteRecordStatus(int? id)
         {
-            var result = await _wasteService.GetWasteRecordStatus(journeyId);
+            if (id == null)
+                return NotFound();
+
+            var result = await _wasteService.GetWasteRecordStatus(id.Value);
+
+            result.WasteRecordReferenceNumber = "WR00012";
```

Comment:
Even though we don't know how this is generated at the moment, it should be set in the service layer, not the Controller layer.

Reply:
Done.